### DISCUSSION TITLE
[ FF7 ] "Oops! Something very bad happened."

### DISCUSSION THREAD

**Discussion by NaoOrLater** [julianxhokaxhiu/FFNx#175] [General]
Hello! 
This is my first time modding and playing a modded game and also first time posting on GitHub so apologies if this is the wrong place to put my issue. I have all of the Qhimm Modding Community mods and Bonez' Modern Spell Names installed on 7th Heaven v2.2.3.522. The game crashes after I select New Game with the error code. I have provided the FFN.LOG below. I can't upload the crash.ff7 file, not sure how to go around that. Appreciate the help! Thanks!
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/5898832/FFNx.log)

**Comment by julianxhokaxhiu** [comment]
As written on the issue screen, before creating one, no 7th support is provided here. Feel free to reach the official forums or Discord for such issues :)
